Library/Formula/cmake.rb

```diff
@@ -3,14 +3,18 @@ class Cmake < Formula
   homepage "http://www.cmake.org/"
   url "https://cmake.org/files/v3.9/cmake-3.9.6.tar.gz"
   sha256 "7410851a783a41b521214ad987bb534a7e4a65e059651a2514e6ebfc8f46b218"
-  revision 1
+  revision 2
```

Comment:
Do we need to revbump it? This is a build error, and I'm not clear if this breaks using curl at runtime.